docs/Operators.md

```diff
@@ -33047,7 +33047,7 @@ expect(node, inputs=[x, repeats], outputs=[z], name="test_tile_precomputed")
 ### <a name="TopK"></a><a name="topk">**TopK**</a>
 
   Retrieve the top-K largest or smallest elements along a specified axis. Given an input tensor of
-  shape [a_1, a_2, ..., a_n, r] and integer argument k, return two outputs:
+  shape [a_1, a_2, ..., a_n] and integer argument k, return two outputs:
```

Comment:
The documentation needs to be be fixed as well in cc files. You may look for `and integer argument k, return two outputs` in the code to find all the location this string appears.

Reply:
Yes, I find this typo appears in many files. Do I need to fix them all?
![image](https://github.com/onnx/onnx/assets/95006218/ee22348f-0403-42fc-bfc5-75bf84295761)

Reply:
Those files are auto-generated. The sources need to be fixed: other than the `defs.cc` file already updated, we need to update the `old.cc` file in the same directory for older opset versions of the op.

Eg [this line](https://github.com/onnx/onnx/blob/75671b3a4f72ae572c664287a14aaabf3f2b81c8/onnx/defs/math/old.cc#L2634) and [this line](https://github.com/onnx/onnx/blob/75671b3a4f72ae572c664287a14aaabf3f2b81c8/onnx/defs/math/old.cc#L2700)

Reply:
Just to add to Xavier's comments: the CI has a mechanism for automatically generate the documentation (using the `auto update doc` label), but it seems to be not working right now due to some git-issue that needs to be fixed.

Reply:
> Just to add to Xavier's comments: the CI has a mechanism for automatically generate the documentation (using the `auto update doc` label), but it seems to be not working right now due to some git-issue that needs to be fixed.

1